.gitignore

```diff
@@ -16,3 +16,4 @@ mkmf.log
 .DS_Store
 .idea/*
 *.gem
+/vendor/
```

Comment:
probably better to use `vendor/*` as this will catch anything with vendor in its name (not that I can think of a great real use case that this blocks).